packages/fx-core/src/component/coordinator/index.ts

```diff
@@ -187,7 +188,12 @@ class Coordinator {
       // refactored generator
       const generator = Generators.find((g) => g.activate(context, inputs));
       if (!generator) {
-        return err(new MissingRequiredInputError(QuestionNames.Capabilities, "coordinator"));
+        return err(
+          new InternalError(
+            new Error(`Unable to activate any generator by inputs: ${JSON.stringify(inputs)}`),
```

Comment:
the error message may show this is an inner error.
so user can figure out it and raise bugs?

Reply:
Updated to a SystemError with message: No activated generator by inputs: xxx. That is helpful for us to trouble shooting.